cmd/img2h/main.go

```diff
@@ -0,0 +1,102 @@
+// Copyright (C) 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// img2h is a utility program that creates C++ headers containing image data.
+// It is usefull to embed texture data within a binary to simplify loading.
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"image"
+	"image/color"
+	"os"
+
+	"github.com/google/gapid/core/app"
+	"github.com/google/gapid/core/log"
+
+	// Import to register image formats.
+	_ "image/gif"
+	_ "image/jpeg"
+	_ "image/png"
+)
+
+var (
+	out = flag.String("out", "-", "Output file, '-' for stdout")
+)
+
+func main() {
+	app.ShortHelp = "img2h generates C++ headers containing image data"
```

Comment:
I would clarify that the header is Vulkan specific:
`img2h generates C++ headers containing Vulkan image data`